data/overlays/gamestates/z_file_nameset_PAL.rodata.s

```diff
@@ -0,0 +1,156 @@
+.include "macro.inc"
+
+ # assembler directives
+ .set noat      # allow manual use of $at
+ .set noreorder # don't insert nops after branches
+ .set gp=64     # allow use of 64-bit general purposee registers
+
+.section .rodata
+
+glabel D_80812A60
+    .asciz "connect_alpha=%d  decision_alpha[%d]=%d\n\x00\x00\x00\x00"
+    .balign 4
+
+.balign 4
```

Comment:
Those 0s are another file split. And that balign is redundant.

Reply:
I can't really figure out at where the file is actually split (looking at what the functions do doesn't really help here). I guess I'll leave it like this for now